tools/c7n_azure/tests/test_event_hub.py

```diff
@@ -40,3 +40,105 @@ def test_find_by_name(self):
         })
         resources = p.run()
         self.assertEqual(len(resources), 1)
+
+    @arm_template('eventhub.json')
```

Comment:
New feature we should use, add `@cassette_name('firewall')` decorator on all these as they all have identical cassettes, then you'll only have 1 cassette for all these tests instead of a bunch of duplicates.

If you don't want to record again just rename one of the cassettes to `EventHubTest.firewall.json` after adding the decorators and delete the rest, then check if they still pass.